owncloudApp/src/main/res/values/strings.xml

```diff
@@ -818,6 +820,10 @@
     <string name="content_description_sort_by_name_ascending">Sort by %1$s ascending</string>
     <string name="content_description_sort_by_name_descending">Sort by %1$s descending</string>
     <string name="content_description_create_new_folder">Create new folder</string>
+    <string name="content_description_space_image">Space image</string>
+    <string name="content_description_space_name">Space name</string>
+    <string name="content_description_space_three_dot_menu">Space three-dot menu</string>
```

Comment:
For a visual impaired person is not relevant which is the menu-format. I'd remove the "three-dot" and let "Space menu" for the talkback